pymc/distributions/distribution.py

```diff
@@ -83,6 +85,21 @@
 PLATFORM = sys.platform
 
 
+class MomentRewrite(GraphRewriter):
+    def add_requirements(self, fgraph):
+        fgraph.attach_feature(ReplaceValidate())
+
+    def apply(self, fgraph):
+        for node in fgraph.toposort():
+            for i, inp in enumerate(node.inputs):
+                if (
+                    hasattr(inp, "owner")
+                    and hasattr(inp.owner, "op")
+                    and isinstance(inp.owner.op, Distribution)
```

Comment:
Maybe we should check if it's a RandomVariable.

We also need to check if an Op had an InnerGraph (like Scan) and do the replacements inside

Reply:
Thank you @ricardoV94, will do it.